src/components/AutoDeposit/AutoDepositSignUpForm.vue

```diff
@@ -0,0 +1,457 @@
+<template>
+	<form
+		class="ad-sign-up-form"
+		@submit.prevent
+		novalidate
+	>
+		<!-- Amount and Day Input -->
+		<div class="row column align-middle">
+			<span class="ad-sign-up-form__deposit-text">
+				Deposit
+			</span>
+			<label
+				class="show-for-sr"
+				:class="{ 'error': $v.adAmount.$invalid }"
+				for="amount"
+			>
+				Amount
+			</label>
+			<kv-currency-input class="text-input" id="amount" v-model="adAmount" />
+			<span class="ad-sign-up-form__deposit-text">
+				each month on the
+			</span>
+			<label class="show-for-sr" :class="{ 'error': $v.$invalid }" :for="dayOfMonth">
+				Day of the Month
+			</label>
+			<input v-if="isDayInputShown"
+				@blur="hideDayInput()"
+				class="text-input"
+				id="dayOfMonth"
+				type="number"
+				placeholder=""
+				required
+				min="1"
+				max="31"
+				v-model.number="dayOfMonth"
+			>
+			<button
+				class="ad-sign-up-form__button--ordinal-day"
+				@click="isDayInputShown = true"
+				v-if="!isDayInputShown"
+			>
+				<strong>{{ dayOfMonth | numeral('Oo') }}</strong>
+				<icon-pencil class="icon-pencil" />
```

Comment:
The end result will be pretty much the same but I think you can use` <kv-icon name="pencil">` for this

Reply:
Good catch, I used this in a couple of places. I'll fix in a follow up PR.